packages/studio-base/src/AppSetting.ts

```diff
@@ -16,7 +16,6 @@ export enum AppSetting {
   ENABLE_NEW_TOPNAV = "enableNewTopNav",
 
   // Privacy
-  TELEMETRY_ENABLED = "telemetry.telemetryEnabled",
```

Comment:
We would like to understand the reason why youre removing telemetry. Can you please elaborate?

Reply:
I assumed that telemetry was already removed, as I was not able to find code that do the telemetry.

Reply:
I am putting it back :-)